doc/source/WorkingWithThePythonScriptingAPI/TutorialStartupConfig1/screengrab.py

```diff
@@ -41,10 +41,7 @@ def __dispatchScript( script, tasks, settings ) :
 		" ".join( settings ),
 		__temporaryDirectory
 		)
-	process = subprocess.Popen( command, shell=True, stderr = subprocess.PIPE )
-	process.wait()
-
-	return process
+	process = subprocess.check_call( command, shell=True )
```

Comment:
Unused variable

Reply:
Ahh darn it, fixed [here](https://github.com/GafferHQ/gaffer/pull/3898/commits/4ed4108f8a7a46425c3a4a5de3bf8c637988d42a), thanks.